### PR TITLE
chore(node-analyzer,kspm-collector): Bump KSPM to 1.38.0 & Node Analyzer to 1.36.0

### PR DESCRIPTION
**Pull request by ghost**
## What this PR does / why we need it:

* Enable NATS continuous reconnect by default for KSPM & Node Analyzer agents.
* Bump KSPM to 1.38.0 and Node Analyzer to 1.36.0

## Checklist

[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]

- [x] Title of the PR starts with type and scope, (e.g. feat(agent,node-analyzer,sysdig-deploy):)
- [x] Chart Version bumped for the respective charts
- [ ] Variables are documented in the README.md (or README.tpl in some charts)
- [ ] Check GithubAction checks (like lint) to avoid merge-check stoppers
- [ ] All test files are added in the tests folder of their respective chart and have a "_test" suffix

Check Contribution guidelines in README.md for more insight.

### CODE REVIEW

File: charts/kspm-collector/Chart.yaml [hunk 1]
```diff
@@ -2,8 +2,8 @@ apiVersion: v2
 name: kspm-collector
 description: Sysdig KSPM collector
 
-version: 0.9.3
-appVersion: 1.35.0
+version: 0.9.4
```
Comment (mavimo) [inline charts/kspm-collector/Chart.yaml]: I suppose this should be a minor and not a patch release

**Review by mavimo — CHANGES_REQUESTED** [review]
The chart bump for the `node-analyzer` is missing

File: charts/kspm-collector/Chart.yaml (hunk 1 above)
Comment (ghost) [reply charts/kspm-collector/Chart.yaml]: @mavimo this is actually just a change to the way an existing ENV flag is enabled. It doesn't add / remove any new functionality.

Comment (mavimo) [reply charts/kspm-collector/Chart.yaml]: @zohar-arad mmm, ok, do you know why the appVersion move from 1.35.0 to 1.36.0 then?

Comment (ghost) [reply charts/kspm-collector/Chart.yaml]: @mavimo sorry - that's my bad. Let me revert it

**Review by chen-shmilovich-sysdig — APPROVED** [review]

File: charts/kspm-collector/Chart.yaml (hunk 1 above)
Comment (ghost) [reply charts/kspm-collector/Chart.yaml]: @mavimo please review and if OK, could you please approve?

**Review by mavimo — APPROVED** [review]
LGTM

**Pull request merged**